psi4/src/psi4/libfock/CMakeLists.txt

```diff
@@ -32,3 +32,14 @@ target_link_libraries(fock
   PRIVATE
     gau2grid::gg
   )
+
+if(TARGET BrianQC::static_wrapper)
+  target_compile_definitions(fock
+    PUBLIC
+      USING_BrianQC
+    )
+  target_include_directories(fock
+    PUBLIC
+      ${BrianQC_INCLUDE_DIRS}
```

Comment:
```suggestion
      $<TARGET_PROPERTY:BrianQC::static_wrapper,INTERFACE_INCLUDE_DIRECTORIES>
```

Reply:
I have changed the include dir syntax to the one you proposed.